src/components/container/FeaturedJobs/FeaturedJobs.scss

```diff
@@ -49,5 +49,6 @@
         display: flex;
         gap: 32px;
         justify-content: space-between;
+        flex-wrap: wrap;
     }
 }
```

Comment:
we might also want to change the padding for the .FeaturedJobs on mobile screens